test/byContract/registries/registies.js.skip

```diff
@@ -1,460 +0,0 @@
-                            '0x1234',
-                            { from: appProvider.address },
-                        );
-
-                        events = tools.extractEvents(
-                            txMined,
-                            AppRegistryInstance.address,
-                            'Transfer',
-                        );
-                        assert.equal(events[0].args.from, constants.NULL.ADDRESS);
-                        assert.equal(events[0].args.to, appProvider.address);
-                        assert.deepEqual(events[0].args.tokenId, web3.utils.toBN(predictedAddress));
-                        AppInstances[i] = await App.at(predictedAddress);
-                    });
-
-                    it('content', async () => {
-                        assert.equal(await AppInstances[i].registry(), AppRegistryInstance.address);
-                        assert.equal(await AppInstances[i].owner(), appProvider.address);
-                        assert.equal(await AppInstances[i].m_appName(), 'App #' + i);
-                        assert.equal(await AppInstances[i].m_appType(), 'DOCKER');
-                        assert.equal(
-                            await AppInstances[i].m_appMultiaddr(),
-                            constants.MULTIADDR_BYTES,
-                        );
-                        assert.equal(
-                            await AppInstances[i].m_appChecksum(),
-                            web3.utils.keccak256('Content of app #' + i),
-                        );
-                        assert.equal(await AppInstances[i].m_appMREnclave(), '0x1234');
-                    });
-
-                    it('token details', async () => {
-                        assert.equal(
-                            await AppRegistryInstance.ownerOf(AppInstances[i].address),
-                            appProvider.address,
-                        );
-                        assert.equal(
-                            await AppRegistryInstance.balanceOf(appProvider.address),
-                            i + 1,
-                        );
-                        assert.isTrue(
-                            await AppRegistryInstance.isRegistered(AppInstances[i].address),
-                        );
-                        assert.equal(
-                            tools.BN2Address(
-                                await AppRegistryInstance.tokenOfOwnerByIndex(
-                                    appProvider.address,
-                                    i,
-                                ),
-                            ),
-                            AppInstances[i].address,
-                        );
-                        assert.equal(
-                            await AppRegistryInstance.tokenURI(AppInstances[i].address),
-                            (await AppRegistryInstance.baseURI()) +
-                                web3.utils.toBN(AppInstances[i].address),
-                        );
```

Comment:
Are these present in your previous PR?

Reply:
No, erc721, I can add them in the same section that I test balanceOf / ownerOf !

Reply:
Done [here](https://github.com/iExecBlockchainComputing/PoCo/pull/124/commits/29701ec3a54bc3bb23e1b5d70b7ce5a9d63532ea)